File: hello.py
import random
from tkinter.font import ROMAN

# 関数定義の練習
def FizzBuzz(sum_number :int) -> str:
    if sum_number%3 == 0 and sum_number%5 == 0:
        return "FizzBuzz"
    elif sum_number%3 == 0:
        return "Fizz"
    elif sum_number%5 == 0:
        return "Buzz"
    else:
        """
         型アノテーションはしているが
         intのまま返してもエラーにはならない
         (その場合普通にint型になる)
        """
        return str(sum_number)

def main():
    print("Hello World!")

    # 要素数5で初期化するがlist型は可変長配列
    list_number = [0 for _ in range(5)]

    for i in range(5):
        list_number[i] = random.randint(0,100)
    
    sum_number = sum(list_number)

    res = FizzBuzz(sum_number)

    print(res)


if __name__ == "__main__" :
    main()

File: hello.java
/*
Javaファイルをコンパイルすると
クラスファイル(JVM用のバイナリ)ができる
これをJava仮想環境で動かす
*/

/*
クラス名とファイル名は同一にする
1ファイルに1クラスを記述する
予約されたクラス名を使用してはいけない
*/

import java.util.Random;
import java.util.stream.IntStream;

public class hello{
    /*
    staticメソッドはインスタンス化しなくても実行できる
    mainメソッドは最初に動く部分なので必ずstatic
    */
    public static void main(String[] args) {
        System.out.println("Hello World!");

        // 要素数5の配列を値0で初期化
        int[] array_number = new int[5];

        // randomクラスのインスタンス化
        Random rand = new Random();

        for (int i=0; i<5; i++) {
            array_number[i] = rand.nextInt(101);
        }

        int sum_number = IntStream.of(array_number).sum();
        // 「文字列型のデータ」ではなくStringクラスのインスタンス
        String res = FizzBuzz(sum_number);
        System.out.println(res);

    }

    // 関数(メソッド)宣言の練習
    private static String FizzBuzz(int sum_number) {
        if (sum_number%3 == 0 && sum_number%5 == 0){
            return "FizzBuzz";
        } else if (sum_number%3 == 0) {
            return "Fizz";
        } else if (sum_number%5 == 0) {
            return "Buzz";
        } else {
            return String.valueOf(sum_number);
        }
    }

}